test/integration/stamp_editor_spec.mjs

```diff
@@ -1814,4 +1814,35 @@ describe("Stamp Editor", () => {
       );
     });
   });
+
+  describe("Switch to edit mode by double clicking on an existing stamp annotation", () => {
+    let pages;
+
+    beforeAll(async () => {
+      pages = await loadAndWait(
+        "issue19239.pdf",
+        getAnnotationSelector("999R")
+      );
+    });
+
+    afterAll(async () => {
+      await closePages(pages);
+    });
+
+    it("must switch to edit mode", async () => {
+      await Promise.all(
+        pages.map(async ([, page]) => {
+          await page.waitForSelector(getAnnotationSelector("999R"));
```

Comment:
Let's define `const annotationSelector = getAnnotationSelector("999R");` once and use that instead of repeating the function call a few times. We try to use this pattern to have the selector definition at a single place, which makes the code a bit shorter/easier to read, avoids repeated lookups and makes it easier to update the test if the selector changes (or when copying and editing existing tests because the body mostly remains the same).